Food-Feedback/Services/RegisterService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using FoodFeedback.Models;
+using FoodFeedback.Services.Interface;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FoodFeedback.Services
+{
+    public class RegisterService : IRegisterService
+    {
+        public bool AddUsers(UserDto userDto)
+        {
+            using (var entity = new FoodFeedbackContext())
+            {
+                var userDetails = new UserRegistration()
```

Comment:
Exception handling is missing in most of the business logic, Do implement the same